purchase_analytic/README.rst

```diff
@@ -0,0 +1,125 @@
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:29e780dfe7855cc5a36aa1348802e5d1035bffb39eb3825103ed2a4fa61f4ada
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--analytic-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-analytic/tree/17.0/purchase_analytic
+    :alt: OCA/account-analytic
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-analytic-17-0/account-analytic-17-0-purchase_analytic
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/builds?repo=OCA/account-analytic&target_branch=17.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+The goal of this module is to ease analytic distribution management on
+purchase order. This module add analytic distribution on purchase order.
+
+If all lines of the purchase order have the same analytic distribution,
+the analytic distribution on the purchase order is automatically set
+with this value. If a analytic distribution is set on the purchase
+order, all lines of the purchase will take this value.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Changelog
+=========
+
+16.0.1.0.0 (2023-01-16)
+-----------------------
+
+Migrated to odoo 16.
+
+15.0.1.0.0 (2022-05-17)
+-----------------------
+
+Migrated to odoo 15.
+
+13.0.1.0.0 (2020-01-08)
+-----------------------
+
+Migrated to odoo 13.
+
+12.0.1.0.0 (2019-10-23)
+-----------------------
+
+Migrated to odoo 12.
+
+10.0.1.0.0 (2017-05-11)
+-----------------------
+
+Migrated to odoo 10.
+
+8.0.1.0.0 (2016-04-22)
+----------------------
+
+First version.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/account-analytic/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us to smash it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/account-analytic/issues/new?body=module:%20purchase_analytic%0Aversion:%2017.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+-------
+
+* Acsone SA/NV
+
+Contributors
+------------
+
+-  Laetitia Gangloff <laetitia.gangloff@acsone.eu>
```

Comment:
Isn't fixing double spaces part of the changes performed by precommit?

I mean:

```suggestion
- Laetitia Gangloff <laetitia.gangloff@acsone.eu>
```